.gitignore

```diff
@@ -0,0 +1,3 @@
+node-operator
+TODO
+!vendor/**
```

Comment:
I took the opportunity to sneak this in.

Reply:
`!vendor/**` is a good catch!

Reply:
What does it mean? :P

Reply:
Ignored patterns don't match under `vendor`:
> An optional prefix "!" which negates the pattern; any matching file excluded by a previous pattern will become included again. It is not possible to re-include a file if a parent directory of that file is excluded. Git doesn’t list excluded directories for performance reasons, so any patterns on contained files have no effect, no matter where they are defined. Put a backslash ("\") in front of the first "!" for patterns that begin with a literal "!", for example, "\!important!.txt".